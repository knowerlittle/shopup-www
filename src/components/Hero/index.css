.heroContainer{
	height: 80vh;
	display: flex;
	flex-direction: row;
	justify-content: center;
	align-items: center;
	background: red;
	margin-top: 8vh;
}

.featureContainer {
	height: 50vh;
	display: flex;
	justify-content: center;
	align-items: center;
	background: grey;
}

.leftSection {
	height: 100%;
	width: 50%;
	display: flex;
	flex-direction: column;
	justify-content: center;
	padding-left: 5%;

	p {
		margin-top: 0em;
		margin-bottom: 0em;
	}
}

.rightSection {
	height: 100%;
	width: 50%;
	display: flex;
	flex-direction: column;
	align-content: center;
	justify-content: center;
	margin-right: 5%;
}

.introText {
	color: #FFFFFF;
	font-size: 14px;
	font-weight: 500;
	letter-spacing: 2px;
}

.mainText {
	font-size: 5vw;
	font-weight: 500;
	line-height: 96px;
	color: #FFFFFF;
	width: 100%;
}

.mainTextLast {
	font-size: 5vw;
	font-weight: 500;
	line-height: 96px;
	color: #FFFFFF;
	width: 100%;
}

.buttonWrapper {
	margin-top: 30px;
}

.videoWrapper {
	width: 100%;
	height: 60%;
}