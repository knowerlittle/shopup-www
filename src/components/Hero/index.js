import React from 'react';
import Nav from '../Nav'
import { Button, Segment, Container, Header } from 'semantic-ui-react';

const Hero = () => (
  	<Segment textAlign='center' inverted style={{minHeight: '700px', padding: '1em 0em'}} vertical>
		<Nav inverted/>
		<Container text>
			<Header as='h1' inverted content=' Discover space for you and your brand' style={ { fontSize: '3em', fontWeight:
			  'normal', margin: '4em 0 2em' }} />
			<Button primary size='huge'>
				Get Started
			</Button>
		</Container>
	</Segment>
)

export default Hero;