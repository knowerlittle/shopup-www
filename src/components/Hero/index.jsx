import React from 'react';
import Button from 'components/Global/Button';
import styles from 'components/Hero/index.css';
import { Link } from 'react-router-dom';
import NeonLike from 'assets/img/home/neonlike.png';

const introText = 'THE INTERNET SAYS IT\'S';

const Hero = () => (
  <div>
    <section className={styles.heroContainer}>
      <div className={styles.leftSection}>
        <p className={styles.introText}>{introText}</p>
        <p className={styles.mainText}>The best way</p>
        <p className={styles.mainText}>to find space</p>
        <p className={styles.mainTextLast}>for your passion.</p>
        <div className={styles.buttonWrapper}>
          <Link to="/signup">
            <Button
              color="purple"
              text="GET STARTED"
              height="50"
              width="300"
            />
          </Link>
        </div>
      </div>
      <div className={styles.rightSection}>
        <img src={NeonLike} alt="like" />
      </div>
    </section>
    <section className={styles.featureContainer}>
      <h2>Feature Container</h2>
    </section>
  </div>
);

export default Hero;
