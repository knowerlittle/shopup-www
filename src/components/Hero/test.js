import React from 'react';
import {shallow} from 'enzyme';
import toJson from 'enzyme-to-json';
import Hero from './';

test('<Hero /> renders correctly', () => {
  const tree = shallow(
    <Hero/>
  );
  expect(toJson(tree)).toMatchSnapshot();
});