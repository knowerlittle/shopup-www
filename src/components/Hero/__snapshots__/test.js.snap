// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Hero /> renders correctly 1`] = `
<div
  className="hero-container"
>
  <h1>
    Home page title
  </h1>
  <button>
    <NavLink
      activeClassName="current"
      ariaCurrent="true"
      to="/signup"
    >
      Get Started
    </NavLink>
  </button>
</div>
`;
